dataminer/Functions/Dashboards_and_Low_Code_Apps/Dashboards_app/Dashboard_Gateway_installation.md

```diff
@@ -84,7 +84,19 @@ There are two main reasons to consider a Dashboard Gateway setup:
    - *connectionString*: The hostname or IP address of the DataMiner Agent to which the Dashboard Gateway has to connect.
    - *connectionUser* and *connectionPassword*: The DataMiner user account that the Dashboard Gateway has to use to connect to the DataMiner Agent (username and password).
 
-1. If [external authentication via SAML](xref:Configuring_external_authentication_via_an_identity_provider_using_SAML) is used, also configure the URL of the API of the Dashboard Gateway (`https://gateway.mycompany.com/API/`) as an *AssertionConsumerService* in the metadata XML file and on the identity provider.
+1. If [external authentication via **SAML**](xref:Configuring_external_authentication_via_an_identity_provider_using_SAML) is used, also configure the URL of the API of the Dashboard Gateway (`https://gateway.mycompany.com/API/`) as an *AssertionConsumerService* in the metadata XML file and on the identity provider.
+
+1. From DataMiner 10.5.0 [CU11]/10.6.2 onwards<!-- RN 44344 -->, to be able to use the **GQI DxM** on the Dashboard Gateway, edit the *web.config* in the API folder, and specify the following settings:
```

Comment:
I would leave out "to be able to use the **GQI DxM** on the Dashboard Gateway" - these settings should now always be configured when setting up (or updating) a Dashboard Gateway